### DISCUSSION TITLE
Prevent user from exiting fullscreen

### DISCUSSION THREAD

**Discussion by wangzhuoranivy** [jspsych/jsPsych#909] [Q&A] [answered]
Hi! I'm wondering if there's a way to prevent user from exiting fullscreen mode with ESC key. Is it possible to change some code in the plugin and exit fullscreen with another key? Or is it possible to track the esc key press in an image-button-response experiment? Thanks a lot.

**Comment by jodeleeuw** [comment, marked as answer]
Hi @wangzhuoranivy ,

There's no way (that I am aware of) to prevent default browser behavior with fullscreen mode. This makes sense as a security/annoyance-prevention feature. If websites could launch into fullscreen mode and then change what keys got out of fullscreen mode it could lead to some pretty malicious and annoying sites.

You can track when users leave fullscreen mode. [See the docs here](https://www.jspsych.org/overview/record-browser-interactions/).

**Reply by wangzhuoranivy** [reply]
Hi Josh. Thank you for the info! That solves my question.
